java/arcs/core/data/CreateableStorageKey.kt

```diff
@@ -0,0 +1,14 @@
+package arcs.core.data
+
+import arcs.core.storage.StorageKey
+
+/**
+ * This class represents a storage key in a compiled [Plan] with 'create' fate.
+ */
+data class CreateableStorageKey(val nameFromManifest: String) : StorageKey("create") {
```

Comment:
`StorageKey` ctor parameter is a `protocol` ("ramdisk" or "db"), not `fate`.. I think somewhere here you'd need to start using `CapabilitiesResolver`.